safe/gis/raster/test/test_zonal_statistics.py

```diff
@@ -8,7 +8,8 @@
 )
 QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
 
-from qgis.core import QGis
+from qgis.core import QGis, QgsCoordinateReferenceSystem
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->